src/commands/build.ts

```diff
@@ -39,10 +39,19 @@ export class BuildCommand implements Command {
           'requests needed to serve your application.'
     },
     {
-      name: 'sw-precache-config',
-      defaultValue: 'sw-precache-config.js',
-      description: 'Path to an sw-precache configuration to be ' +
-          'used for service worker generation.'
+      name: 'add-service-worker',
```

Comment:
until we support more than sw-precache, we may want to name this something sw-precache specific. maybe `--add-sw-precache` or `--use-sw-precache`